### PR TITLE
Bug 1872405: Add requested capacity to pvc details page

### PR DESCRIPTION
**Pull request by cloudbehl**
Screenshot 2020-08-25 at 9 47 57 PM

Signed-off-by: Ankush Behl <cloudbehl@gmail.com>

### CODE REVIEW

**Comment by cloudbehl** [comment]
@spadgett can you have a look.

File: frontend/public/components/persistent-volume-claim.jsx [hunk 1]
```diff
@@ -265,6 +274,10 @@ const Details_ = ({ flags, obj: pvc }) => {
               <dd data-test-id="pvc-status">
                 <PVCStatus pvc={pvc} />
               </dd>
+              <dt>Requested Capacity</dt>
+              <dd data-test="pvc-requested-capacity">
```
Comment (spadgett) [inline frontend/public/components/persistent-volume-claim.jsx:278]: Is this being used in a test? I'd only add the data-test attribute when we plan to use it.

Comment (cloudbehl) [reply frontend/public/components/persistent-volume-claim.jsx:278]: Yes, This is required for the clone testing.

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**